src/app/ngrx/reducers/episode.reducer.ts

```diff
@@ -69,7 +69,11 @@ export function reducer(
       };
     }
     case ActionTypes.CASTLE_EPISODE_PAGE_FAILURE: {
-      return {...state, error: action.payload.error};
+      return {
+        ...state,
+        error: action.payload.error,
```

Comment:
Ah, so does this mean you're passing remote errors around in the application state, and reacting to that to show some UI?